doc/install/configure_ide.rst

```diff
@@ -0,0 +1,48 @@
+.. _configure-ide:
+
+IDE Configuration Guide
+=======================
+
+.. include:: ../links.inc
+.. highlight:: console
+
+Thanks for taking the time to contribute! This guide steps through various IDE settings that are helpful when
+contributing to MNE-Python.
```

Comment:
```suggestion
This guide steps through various IDE settings that are helpful when contributing to MNE-Python.
```
Given that this will land somewhere within the contributor's guide, I don't think we will need to add the thank you again.